src/recommendationservice/recommendation_server.py

```diff
@@ -66,12 +59,34 @@ def Watch(self, request, context):
         return health_pb2.HealthCheckResponse(
             status=health_pb2.HealthCheckResponse.UNIMPLEMENTED)
 
+
+def get_product_list(request_product_ids):
+    with tracer.start_as_current_span("get_product_list") as span:
+        max_responses = 5
+        # fetch list of products from product catalog stub
+        cat_response = product_catalog_stub.ListProducts(demo_pb2.Empty())
+        product_ids = [x.id for x in cat_response.products]
+        span.set_attribute("app.products.count", len(product_ids))
```

Comment:
Probably want to hold off on merging this until we resolve the service name inclusion / attribute naming conventions in the other pr